twitter.html

```diff
@@ -0,0 +1,275 @@
+                     <h4 class="clara">Clara</h4> <span>&#x40;Claroquesi · 30m</span>
```

Comment:
I don't understand the need for specific user classes – you seem to only use the for the display of the usernames?